pkg/golinters/godox.go

```diff
@@ -0,0 +1,46 @@
+package golinters
+
+import (
+	"context"
+	"go/token"
+	"strings"
+
+	"github.com/golangci/golangci-lint/pkg/lint/linter"
+	"github.com/golangci/golangci-lint/pkg/result"
+
+	"github.com/matoous/godox"
```

Comment:
I see `https://github.com/matoous/godox` is a fork. Can you please explain in your fork's README reasoning for forking?

Reply:
Updated! Good point.